components/core/src/ffi/encoding_methods.hpp

```diff
@@ -249,6 +249,35 @@ namespace ffi {
             encoded_variable_t* encoded_vars,
             int encoded_vars_length
     );
+
+    /**
+     * Checks whether the given wildcard strings match the given encoded
+     * variables (from a message). Specifically, let {w in W} be the set of
+     * wildcard strings and {e in E} be the set of encoded variables. This
+     * method will return true only if:
+     * (1) Each unique `w` matches a unique `e`.
+     * (2) When (1) is true, the order of elements in both W and E is unchanged.
+     * @tparam encoded_variable_t Type of the encoded variable
+     * @param logtype The message's logtype
+     * @param encoded_vars The message's encoded variables
+     * @param encoded_vars_length
```

Comment:
can we be clear on what length mean? does it mean number of encoded_vars or size of the encoded_var buffer?

Reply:
Clarified